packages/server/modules/core/services/branch/management.ts

```diff
@@ -45,7 +45,11 @@ export async function createBranchAndNotify(
 
   const branch = await createBranch({
     name: input.name,
-    description: isBranchCreateInput(input) ? input.description || null : null,
+    description: isBranchCreateInput(input)
```

Comment:
I think since both types have the description field you can just shorten all of this to:

```
description: input.description ?? null
```